addon/components/plugins/list/ordered.hbs

```diff
@@ -2,22 +2,37 @@
 {{#if @controller}}
   <Toolbar::Button
     @active={{this.isActive}}
-    @title={{t 'ember-rdfa-editor.ordered-list.button-label'}}
-    @optionsLabel={{t 'ember-rdfa-editor.ordered-list.options-label'}}
+    @title={{t "ember-rdfa-editor.ordered-list.button-label"}}
```

Comment:
Is there a preference to use `"` in place of `'`? Our template lint config doesn't give a preference (at least it doesn't error for me if I run it on a file with a mixture) but for the `gts` components, prettier starts looking at them and judges us if we use `"` (other than if there are e.g. nested quotes). We can configure this differently, but we probably shouldn't each convert files in the opposite direction...

Reply:
I'm more used to seeing double quotes in html-like languages, but I really don't have any preferences
You're right that we should configure everything to match though. Lets look at what the ember default setting is and go with that